src/components/ConsortiaControlledVocabulary/ConsortiaControlledVocabulary.js

```diff
@@ -102,6 +102,8 @@ export const ConsortiaControlledVocabulary = ({
   const eventEmitter = useEventEmitter();
   const [activeDialog, setActiveDialog] = useState(null);
 
+  console.log(props)
```

Comment:
need to remove it. The rest looks good.

Reply:
yep, I know, it's to check on env - I don't have all modules locally